lib/openglvis.hpp

```diff
@@ -195,7 +195,8 @@ class VisualizationScene
    glm::mat4 rotmat;
    glm::mat4 translmat;
 
-   float matAlpha = 1.0;
+   // Needs to be < 1.0 to allow alpha defined at palette level to show through
+   float matAlpha = 0.999999;
```

Comment:
Why this? I thought they are multiplied 🤔

Reply:
I don't understand it enough yet to explain why 😅 . When it is 1 though, all the alpha values set by the discrete/smooth texture for each palette are ignored.

Reply:
There is a check `param.is_translucent = matAlpha < 1.0` in several areas. I added a check in [006a042](https://github.com/GLVis/glvis/pull/289/commits/006a042df35e21ad105daa22bc539a77f14da90c) so it shouldn't be necessary anymore.